rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -539,6 +539,52 @@ void setLockWhitelistDisableBlockDelay() throws BlockStoreException, IOException
         }
     }
 
+    @Nested
+    @Tag("LockingCap")
+    class LockingCapTest {
+
+        private LockingCapSupport lockingCapSupport;
+        private BridgeSupport bridgeSupport;
+        LockingCapConstants constants = LockingCapMainNetConstants.getInstance();
+
+        @BeforeEach
+        void setUp() {
+            lockingCapSupport = mock(LockingCapSupportImpl.class);
+            bridgeSupport = bridgeSupportBuilder
+                .withLockingCapSupport(lockingCapSupport)
+                .build();
+        }
+
+        @Test
+        void getLockingCap() {
+            // Arrange
+            Optional<Coin> expectedLockingCap = Optional.of(constants.getInitialValue());
+            when(lockingCapSupport.getLockingCap()).thenReturn(expectedLockingCap);
+
+            // Act
+            Optional<Coin> actualLockingCap = Optional.of(bridgeSupport.getLockingCap());
+
+            // Assert
+            assertEquals(expectedLockingCap, actualLockingCap);
```

Comment:
```
      // Act
      Coin actualLockingCap = bridgeSupport.getLockingCap();

      // Assert
      assertEquals(expectedLockingCap.get(), actualLockingCap);
```

to me, it sounds weird that the actualValue is wrapped with an Optional

Reply:
another option, following what you did on the test below:

```
      // Arrange
      Coin expectedLockingCap = constants.getInitialValue();
      when(lockingCapSupport.getLockingCap()).thenReturn(Optional.of(expectedLockingCap));
      
      // Act
      Coin actualLockingCap = bridgeSupport.getLockingCap();

      // Assert
      assertEquals(expectedLockingCap, actualLockingCap);
```

Reply:
At first glance it was weird to me as well.

Reply:
We are handling `Optional` results from methods. Typically in a test, you can handle it how it is to avoid validating `isPresent()` calls and avoiding adding more code lines if not necessary. WDYT? @nathanieliov @julia-zack

Reply:
No prob. LGTM